test/e2e/basic_install/install.go

```diff
@@ -18,132 +18,61 @@ package basic_install
 
 import (
 	"context"
+	"log"
 	"time"
 
 	"github.com/onsi/ginkgo"
 	"github.com/onsi/gomega"
-	machineconfigv1 "github.com/openshift/machine-config-operator/pkg/apis/machineconfiguration.openshift.io/v1"
-	machineconfigclientset "github.com/openshift/machine-config-operator/pkg/generated/clientset/versioned/typed/machineconfiguration.openshift.io/v1"
-	"k8s.io/apimachinery/pkg/api/errors"
+
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	"k8s.io/kubernetes/test/e2e/framework"
+	"sigs.k8s.io/controller-runtime/pkg/client"
```

Comment:
previously, we were using the k8s framework to enable the import of the RTE testsuite, which is based on k8s framework (in turn to maximize the interoparability with the NFD testsuite)

I still like consuming the rte e2e suite as separate binary (!) so let's keep this direction